grpc/src/main/java/io/stargate/grpc/service/Service.java

```diff
@@ -94,38 +134,47 @@ private void handleError(Throwable throwable, StreamObserver<Result> responseObs
     if (throwable instanceof StatusException || throwable instanceof StatusRuntimeException) {
       responseObserver.onError(throwable);
     } else {
-      responseObserver.onError(Status.UNKNOWN.withCause(throwable).asRuntimeException());
+      responseObserver.onError(
+          Status.UNKNOWN
+              .withDescription(throwable.getMessage())
```

Comment:
Unfortunately, It doesn't get to the client. I just put message in there for debugging right now because that does get to the caller. I have a ticket to fix this. https://github.com/stargate/stargate/issues/923